tests/test_interface_port_channel.rb

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'ciscotest'
+require_relative '../lib/cisco_node_utils/interface'
+require_relative '../lib/cisco_node_utils/interface_port_channel'
+
+# TestX__CLASS_NAME__X - Minitest for X__CLASS_NAME__X node utility class
+class TestInterfacePortChannel < CiscoTestCase
+  # TESTS
+
+  DEFAULT_NAME = 'port-channel134'
+
+  def setup
+    super
+    config "no interface #{DEFAULT_NAME}"
+  end
+
+  def teardown
+    config "no interface #{DEFAULT_NAME}"
+    super
+  end
+
+  def n7k_platform?
+    /N7K/ =~ node.product_id
+  end
+
+  def n9k_platform?
+    /N3K/ =~ node.product_id || /N9K/ =~ node.product_id
+  end
+
+  def n6k_platform?
+    /N5K/ =~ node.product_id || /N6K/ =~ node.product_id
+  end
+
+  def create_port_channel(ifname=DEFAULT_NAME)
+    InterfacePortChannel.new(ifname)
+  end
+
+  def test_get_set_port_hash_distribution
+    return if n6k_platform?
```

Comment:
This should be a skip, e.g.:

`    skip('Platform does not support this property') if n6k_platform?